docs/standard/datetime/choosing-between-datetime.md

```diff
@@ -38,6 +38,30 @@ ms.assetid: 07f17aad-3571-4014-9ef3-b695a86f3800
 > [!NOTE]
 > This topic doesn't discuss <xref:System.TimeZone> because its functionality is almost entirely incorporated in the <xref:System.TimeZoneInfo> class. Whenever possible, use the <xref:System.TimeZoneInfo> class instead of the <xref:System.TimeZone> class.
 
+## The DateTimeOffset structure
+
+The <xref:System.DateTimeOffset> structure represents a date and time value, together with an offset that indicates how much that value differs from UTC. Thus, the value always unambiguously identifies a single point in time.
+
+The <xref:System.DateTimeOffset> type includes all of the functionality of the <xref:System.DateTime> type along with time zone awareness. This makes it suitable for applications that:
+
+- Uniquely and unambiguously identify a single point in time. The <xref:System.DateTimeOffset> type can be used to unambiguously define the meaning of "now", to log transaction times, to log the times of system or application events, and to record file creation and modification times.
+
+- Perform general date and time arithmetic.
+
+- Compliant with ANSI and ISO 8601 standards.
```

Comment:
what exactly you mean by that?

Reply:
I think this will be confusing statement. DateTimeOffset support formatting with 'O' and 'r' for round tripping and RFC1123 which is close to ISO 8601 but not exact. Also, DateTimeFormat support many other format than ISO 8601. https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings.

Reply:
This is why it was still draft and unfortunately fell to the way side. I had hard time with this section.

Reply:
ok, please remove this statement then.

Reply:
Done.